src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Types/RoByRefType.cs

```diff
@@ -25,7 +25,7 @@ internal RoByRefType(RoType elementType)
 
         public sealed override int GetArrayRank() => throw new ArgumentException(SR.Argument_HasToBeArrayClass);
 
-        protected sealed override TypeAttributes ComputeAttributeFlags() => TypeAttributes.AnsiClass;
+        protected sealed override TypeAttributes ComputeAttributeFlags() => TypeAttributes.Public;
```

Comment:
Could you please also change this: https://github.com/dotnet/runtime/blob/b2dc37ba181a7fa4427e717eab819ba3543d0ae4/src/coreclr/nativeaot/System.Private.Reflection.Core/src/System/Reflection/Runtime/TypeInfos/RuntimeHasElementTypeInfo.cs#L162 to return `TypeAttributes.Public`

Reply:
Sure